### ISSUE TITLE
Display slope data in summary and tooltip without graph

### ISSUE THREAD

**Issue by gegeweb** [Raruto/leaflet-elevation#62]
In the new release with the `slope: true`option, is it possible to not display slope graph and have informations only in summary and tooltip?

I think no.

Perhaps slope option should be false for nothing, true for all, and 'summary' (for example) for not displaying the graph but computing informations about elevation gain and loss and slope in %.

**Comment by Raruto** [comment]
Do you mean?

```js
slope: false,     // default
slope: true,      // slope chart + summary
slope: "summary", // only summary (i.e. hide slope chart)

```

Still under development, but here you can see a first working example of what could be the "next version":

https://github.com/Raruto/leaflet-elevation/blob/0749e8cada8b8a1823ec714aedf528590e7a9db0/test/index.html#L69

**Comment by gegeweb** [comment]
Yes, that's exactly it. 👍 

For the summary, works fine.
But now tooltip isn't hidden when leaving chart or layer. And legend always diplayed even if `legend: false`option.

**Comment by Raruto** [comment]
> tooltip isn't hidden when leaving chart or layer. And legend always diplayed even if `legend: false` option.

Now it should be fine, but more tests will be needed ...

**Comment by gegeweb** [comment]
Everything seems to be working properly.
Small patch sent to allow the L.marker to follow the position of the mouse.
Without that, his events (`mousemove`/`mousehover`) take priority over the events of the layer.

**Comment by Raruto** [comment]
Released as **version [1.2.1](https://www.npmjs.com/package/@raruto/leaflet-elevation/v/1.2.1)**

**Comment by gegeweb** [comment]
I've opened a new PR #64 to fix an error when binding external layer(s).
This error wasn't in 1.2.0 and previous commit.
